### PR TITLE
Add EnglishOthers to Python cultures

### PR DESCRIPTION
**Pull request by samhickey25**
Add EnglishOthers to the list of Python cultures which will use the dmy format for dates

### CODE REVIEW

File: Python/libraries/recognizers-date-time/recognizers_date_time/date_time/english/date_extractor_config.py [hunk 1]
```diff
@@ -156,19 +156,38 @@ def written_month_regex(self) -> Pattern:
     def month_suffix_regex(self) -> Pattern:
         return self._month_suffix_regex
 
-    def __init__(self):
+    def __init__(self, dmyDateFormat=False):
         self._check_both_before_after = EnglishDateTime.CheckBothBeforeAfter
+        if dmyDateFormat:
```
Comment (samhickey25) [inline Python/libraries/recognizers-date-time/recognizers_date_time/date_time/english/date_extractor_config.py:161]: The order here is important and is based of what is in .NET (and javascript)

File: Python/libraries/recognizers-text/recognizers_text/culture.py [hunk 2]
```diff
@@ -15,6 +16,42 @@ class Culture:
...
+            Culture.Korean,
+            Culture.Portuguese,
+            Culture.Spanish,
+            Culture.SpanishMexican,
+            Culture.Turkish,
+            Culture.German
+        ]
+
+    @staticmethod
+    def map_to_nearest_language(culture_code: str):
```
Comment (samhickey25) [inline Python/libraries/recognizers-text/recognizers_text/culture.py:37]: Again this is based on an existing function in .NET

**Comment by srandal** [comment]
We'll also want to add a couple of dot-delimited dates to the en-gb test cases. Will that be in a separate PR?

**Review by srandal — APPROVED** [review]

**Review by kevinwalshgen — APPROVED** [review]

**Review by sanhal — APPROVED** [review]

File: Python/libraries/recognizers-date-time/recognizers_date_time/resources/spanish_date_time.py [hunk 3]
```diff
@@ -603,7 +603,9 @@ class SpanishDateTime:
                                  ("^a[nñ]o$", "(?<!el\\s+)a[nñ]o"),
                                  ("^semana$", "(?<!la\\s+)semana"),
                                  ("^mes$", "(?<!el\\s+)mes"),
-                                 ("^(abr|ago|dic|feb|ene|ju[ln]|mar|may|nov|oct|sep?t|sep)$", "([$%£&!?@#])(abr|ago|dic|feb|ene|ju[ln]|mar|may|nov|oct|sep?t|sep)|(abr|ago|dic|feb|ene|ju[ln]|mar|may|nov|oct|sep?t|sep)([$%£&@#])")])
```
[the remark this answers is not in the document]
Comment (samhickey25) [reply Python/libraries/recognizers-date-time/recognizers_date_time/resources/spanish_date_time.py:606]: Ah these are just updates from running the script that updates the python regex with the latest patterns. That script is also always run when we build the recognisers library so may as well keep them

**Pull request merged**